rich/__init__.py

```diff
@@ -11,7 +11,7 @@
     from .console import Console
 
 # Global console used by alternative print
-_console: Optional["Console"] = None
+_console_default_singleton: Optional["Console"] = None
```

Comment:
I stumbled across that Python behaviour I was not aware of: it seems that in this `__init__.py` file the fact that we have a variable named `_console` conflicts with a `_console.py` module if there is one.
So `global _console` will actually refer to the `_console` **module**, rather than this variable! :open_mouth: 

Renaming it avoids the problem.

Screenshot of the issue:
(not caused by the fact that the local variable is also named `_console` , but by the fact that `get_console()` will return the `_console.py` module instead of the global `_console` variable)
![Screenshot from 2022-04-05 15-37-56](https://user-images.githubusercontent.com/722388/161779207-195099be-4e87-4441-98bd-ec77910c99d5.png)